db/texts/event_info.md

```diff
@@ -1,37 +1,53 @@
-## Steel City Ruby 2014 Call for Proposals (CFP)
-Hi! This page is here to provide you some information about our process for gathering, evaluating, and selecting proposals for talks for this year's Steel City Ruby. Our goal is to select&ndash;objectively and transparently as possible&ndash;the best proposals from those submitted for this year's event.
+## Open Source & Feelings 2015 Call for Proposals (CFP)
 
-### What's _best_ mean?
-That is a good question. In the context of this year's event, _best_ means most appealing, useful, and relevant to our attendees. While those terms are subjective, we've modeled our selection process to help eliminate biases and encourage open, honest discussion and evaluation based on the comparative merits of the submitted proposals.
+Greetings! We look forward to getting a wide range of proposals for Open Source & Feelings. The info on this page will help you prepare for submitting your proposal.
 
-### Our Process
-SCRC will be considering _Technical_ and _People/Process_ talk proposals separately this year, formalizing the balance we've had and enjoyed in previous years. A single proposal submission can be included in either or both focus areas, and we will select a balance of both presenters and alternates from each group.
+### About You
 
-The proposal form is divided into four sections, three of which are used as criteria for acceptance. 
+You have feelings. You care about open source. Maybe you have thoughts on how we can work better in our open source communities. Maybe you have thoughts on how our humanity impacts our OS work, or vice versa. Perhaps you've read through our wishlist and something caught your eye. When you heard about this conference you had an idea that you just knew would fit.
 
-_Talk Information_ is the primary data used during proposal evaluation and selection. Please help us to remain as unbiased as possible by not including personally identifying information in this section such as pronouns, websites, or company names. 
+We look forward to wide diversity in our speakers, presentation formats, and topics.
+
+### Topics
+
+We are interested in practical information that our participants can take back with them to their Open Source communities. We are interested in authentic presentations opening up needed conversations in the larger community. Our wishlist may be helpful to get the ideas flowing.
+
+### Proposal Details
+
+Pick a format for your presentation, such as a formal talk, panel, activity, spoken word, etc.
+
+Choose a length for your presentation between 10 and 40 minutes.
```

Comment:
I don't think 10 and 40 were hard limits. Maybe update to something like, "Choose a length for your presentation. Most are between 10 and 40 minutes, but we will consider other options."